dev_guide/quota.adoc

```diff
@@ -11,59 +11,99 @@ toc::[]
 
 == Overview
 
-A *ResourceQuota* object enumerates hard resource usage limits per project. It
-limits the total number of a particular type of object that may be created in
-a project, and the total amount of
-link:compute_resources.html[compute resources]
+A *ResourceQuota* defines constraints that limit aggregrate resource consumption
+per project.  It can limit the quantity of objects that can be created in a project
+by type, as well as the total amount of link:compute_resources.html[compute resources]
 that may be consumed by resources in that project.
 
-== Usage Limits
+== Resources managed by quota
 
-The following describes the set of limits that may be enforced by a
-*ResourceQuota*.
+The following describes the set of resources that may be managed by a *ResourceQuota*.
 
-.Usage limits
+.Resources managed by quota
```

Comment:
Yeah, I was waiting for that to finish.  I think a follow-up for that is
perfect.

On Wed, Mar 16, 2016 at 11:03 AM, Michal Minar <notifications@github.com>
wrote:

> In dev_guide/quota.adoc
> <https://github.com/openshift/openshift-docs/pull/1747#discussion_r56349651>
> :
>
> >
> > -.Usage limits
> > +.Resources managed by quota
>
> There's also openshift.io/images (openshift/origin#7930
> <https://github.com/openshift/origin/pull/7930>). But it hasn't been
> merged yet. I can make a follow-up.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/1747/files/22c1b420197c93f7d938d3ad660fb3a2b8c5a160#r56349651>
>